pkgs/applications/misc/yate/default.nix

```diff
@@ -33,7 +33,7 @@ stdenv.mkDerivation rec {
 
   meta = {
     description = "Yet another telephony engine";
-    homepage = "http://yate.ro/";
+    homepage = "https://yate.ro/";
```

Comment:
Also fix the broken download yate.null.ro and switch src to Github https://github.com/yatevoip/yate